paper/paper.md

```diff
@@ -0,0 +1,512 @@
+ask for help by putting up a colored sticky note (or a Zoom emoji in the case of
+virtual meetings) [@becker_responding_2016].
+1. We exposed the learners to different aspects of data science by bringing in
+women guest speakers from academics and industry. This allowed them to better
+put what they were learning into context and think about how they might use the
+skills they were learning in potential future careers, and exposed them to
+successful women in STEM.
+
+### Learner experiences
+
+We surveyed learners anonymously after each Club and Summer Experience and found
+that they all felt their skills in Python programming,
+problem solving, critical thinking, and collaboration had improved.
+Furthermore, for the 2019 instance of Club we asked that all participants take a
+pre-test and a post-test comprising Python coding questions.
+While only five participants recorded their post-test, all of them answered more
+questions correctly on the post-test than the pre-test (range 1-8 more questions
+correct out of 11).
+
+Overwhelmingly, learners' favorite parts of the program are the guest speakers
+and the project.
+These aspects of our curriculum expose them to new fields and allow them to
+apply their newfound coding skills to asking an interesting question.
+About a third of participants claim that they are now more interested in
+pursuing a career in computer or data science compared to before their Girls Who
+Code experience.
+
+# Acknowledgements
+
+We would like to acknowledge our faculty co-sponsors Maureen Sartor &
+Cristina Mitrea.
+We appreciate the continued support of U-M DCMB staff and faculty including
+Julia Eussen, Mary Freer, Linda Peasley, Jane Wiesner, Brian Athey, and
+Margit Burmeister.
+We are grateful for the resources provided by the national Girls Who Code
+organization.
+
+Our programming is made possible by the dedication of past and present Executive
+Committee members, Club and Summer Experience Facilitators, and Capstone Project
+mentors including Alex Weber, Arushi Varshney, Sophie Hoffman, Hojae Lee, Ruma
+Deb, Saige Rutherford, Michelle McNulty, Bailey Peck, Chloe Whicker, Carolina
+Rojas Ramirez, Verity Sturm, Zoe Drasner, Sarah Latto, Emily Roberts, Angel Chu,
+Vivek Rai, Hillary Miller, Ashton Baker, Murchtricia Jones, Lauren Jepsen,
+Aubrey Annis, Awanti Sambarey, Mengtong Hu, Maribel Okiye, Yingxiao Zhang, and
+Neslihan Bisgin.
+
+Our student organization has received funding from the U-M Department of
+Computational Medicine and Bioinformatics, the U-M Office of Graduate and
+Postdoctoral Studies, anonymous donations from Giving Blue Day 2019, the U-M
+Endowment in Basic Sciences, and the Girls Who Code Support Fund.
+The 2019 Summer Experience was sponsored by Zingerman’s Delicatessen, the U-M
+Statistics Department, the U-M Detroit Center, Dell, Cisco, the U-M Department
+of Computational Medicine and Bioinformatics, and the U-M Biostatistics
+Department.
```

Comment:
Can we make this section more concise?

Also, according to our records, we received funding from U-M ARC and LSI for SE 2019, so I added them in this suggestion.

```suggestion
Our student organization has received funding and other support from 
the U-M Department of Computational Medicine and Bioinformatics, 
the U-M Biostatistics Department, 
the U-M Statistics Department, 
the U-M Office of Graduate and Postdoctoral Studies, 
the U-M Endowment in Basic Sciences, 
the U-M Detroit Center, 
the U-M Life Sciences Institute,
the U-M Advanced Research Computing
anonymous donations from Giving Blue Day 2019,
Zingerman’s Delicatessen, 
Dell, 
Cisco, 
and the Girls Who Code Support Fund.
```